easybuild/easyconfigs/j/Julia/Julia-1.10.3-linux-x86_64.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'Tarball'
+
+name = 'Julia'
+version = '1.10.3'
+versionsuffix = '-linux-x86_64'
+
+homepage = 'https://julialang.org'
+description = "Julia is a high-level, high-performance dynamic programming language for numerical computing"
+
+toolchain = SYSTEM
+
+source_urls = ['https://julialang-s3.julialang.org/bin/linux/x64/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+patches = [('julia.wrapper', 'bin/')]
+checksums = [
+    {'julia-1.10.3-linux-x86_64.tar.gz': '81b910c922fff0e27ae1f256f2cc803db81f3960215281eddd2d484721928c70'},
+    {'julia.wrapper': 'a36a3e0cb98ab4d2b05e0ca8f0bf2724baafbd669a0db318a00e40737d2f02f9'},
```

Comment:
@lexming  it seems the checksum for `julia.wrapper` defined on the easyconfig here is not the correct one.

We are pulling the script from #20103   where all easyconfigs there have:

```    
{'julia.wrapper': 'd10aeaff53cca9875f7b0ce9218eae3bd21870b654e26c4c52aa8bfcc9da702d'},
```

which seems to be the actual sum.

Reply:
thanks @arielzn ! I updated the checksum of the wrapper in https://github.com/easybuilders/easybuild-easyconfigs/pull/20103 and indeed forgot about this PR :slightly_smiling_face: